mobile/node_test.go

```diff
@@ -69,6 +70,17 @@ func TestNode_GetPhotoBase64String(t *testing.T) {
 	}
 }
 
+func TestRecoveryPhrase(t *testing.T) {
+	mnemonic, err := textile.GetRecoveryPhrase()
+	if err != nil {
+		t.Errorf("failed to create new recoveryphrase: %s", err)
+	}
+	list := strings.Split(mnemonic, " ")
+	if len(list) != 24 {
+		t.Errorf("got bad mnemonic length: %s", len(list))
+	}
+}
+
```

Comment:
sweet, could also add a test that tries to recover the priv key from this...
```
seed := bip39.NewSeed(mnemonic, "") // not using a pass here
identityKey, err := repo.IdentityKeyFromSeed(seed, nBitsForKeypair)
```
then would need to unmarshal that byte slice as a priv key and compare, or maybe compare bytes?. also, our `repo` package doesn't currently export `identityKeyFromSeed`... prob better to put this test in repo_test anyway.